### PR TITLE
Set opens="auto" as default to address calendar overflow issue in small browser windows

### PR DESCRIPTION
**Pull request by angelo510**
https://stackblitz.com/edit/ngx-daterangepicker-material-issue

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ngx-daterangepicker-material",
-  "version": "2.1.7",
+  "name": "mr-daterangepicker",
```
Comment (fetrarij) [inline package.json]: - Do not rename the package name
- Do not change the version number

File: package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "moment",
     "picker"
   ],
-  "homepage": "https://github.com/fetrarij/ngx-daterangepicker-material",
+  "homepage": "https://github.com/angelo510/ngx-daterangepicker-material",
```
Comment (fetrarij) [inline package.json]: Do not change homepage

File: package.json [hunk 3]
```diff
@@ -32,10 +32,10 @@
     "gh-pages": "ng build --prod --base-href=\"/ngx-daterangepicker-material/\" && ngh --dir demo/dist",
     "build": "ng-packagr -p package.json"
   },
-  "author": "Fetrarijaona R.",
+  "author": "angelo510",
```
Comment (fetrarij) [inline package.json]: Do not change the author

File: package.json [hunk 4]
```diff
@@ -32,10 +32,10 @@
...
   "repository": {
     "type": "git",
-    "url": "https://github.com/fetrarij/ngx-daterangepicker-material.git"
+    "url": "https://github.com/angelo510/ngx-daterangepicker-material.git"
```
Comment (fetrarij) [inline package.json]: Do not change url

**Comment by angelo510** [comment]
package.json has been reverted.

**Comment by fetrarij** [comment]
@angelo510 thanks for the PR :)

**Comment by angelo510** [comment]
No problem! @fetrarij

**Pull request merged**